cfgov/jinja2/v1/owning-a-home/explore-rates/index.html

```diff
@@ -536,11 +536,19 @@ <h4 class="next-steps-heading">Save for a larger down payment.</h4>
                             block__border-top
                             block__flush-top
                             block__padded-top
-                            note-ctr">
-                    <div class="note">
-                        <p class="alert"><strong>Check your credit report.</strong></p>
-                        If you haven’t checked your credit report recently, <a id="annualcreditreport" class="external-link" href="https://annualcreditreport.com" target="_blank" rel="noopener noreferrer">do so now</a>. If you find errors, <a id="ask-dispute-credit-report" href="/askcfpb/314/how-do-i-dispute-an-error-on-my-credit-report.html" target="_blank" rel="noopener noreferrer">get them corrected</a> <strong>before</strong> you apply for a mortgage.
-                    </div> <!-- /.note -->
+                            credit-note-ctr">
+                    <div class="credit-note
+                                m-notification m-notification__warning
```

Comment:
Let's move `m-notification__warning` to its own line.

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/4100/commits/70e143a4407fbde245f74d2c9023ab588016c378.